lte/gateway/c/core/oai/test/s1ap_task/data/s1ap_state_ATTACHED_encoded

```diff
@@ -0,0 +1 @@
+�����������������������������������������������������������������������������������������ů�����������������������������ѽ�
```

Comment:
Hi @thhung Could you please let us know how is this file generated?

Reply:
HI @rsarwad , I don't have access to the testbed I used to develop so I cannot tell you for sure. From what I remember, this file is generated by applying base64 encoding and after that, I applied a shift to the value of the base64-encoded message. You can take a look at how it is decoded [here](https://github.com/magma/magma/blob/master/lte/gateway/c/core/oai/test/s1ap_task/mock_s1ap_op.cpp#L136). It can give you information on how it is encoded by reversing the process. If you have some questions, let me know. I will try to see if I can get some more info for you. Hope this helps.

Reply:
@ardzoht , can you provide some context here?

Reply:
Thanks, @ardzoht , for helping me clarify the context.

Reply:
Thank you everyone for providing me the context.
Currently I followed the approach, the way it was followed for reading UE context 
I removed the encoded form of s1ap state. Just keeping result of state_cli.py was failing to parse this file. Because file needs to be in binary form.
From sample program, I wrote the s1ap state info file, using  SerializeToOstream() which eventually generates file in binary format. From s1ap test suite, modified code to read generated file.
Please let me know if anyone has any concerns on this approach.